### PR TITLE
aries demo test case

### PR DESCRIPTION
**Pull request by solarw**
## Proposed changes

aries demo test case: fixes, automated test

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

DELETE INCLUSIVE THIS AND BELOW FOR STANDARD PR
------

## Release summary

Version number: [e.g. 1.0.1]

## Release details

Describe in short the main changes with the new release.

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side), from `develop`
- [ ] Lint and unit tests pass locally and in CI
- [ ] I have checked the fingerprint hashes are correct by running (`scripts/generate_ipfs_hashes.py`)
- [ ] I have regenerated the latest API docs
- [ ] I built the documentation and updated it with the latest changes
- [ ] I have added an item in `HISTORY.md` for this release
- [ ] I bumped the version number in the `aea/__version__.py` file.
- [ ] I bumped the version number in every Docker image of the repo and published it. Also, I built and published them with tag `latest`  
      (check the READMEs of [`aea-develop`](../develop-image/README.md#publish) 
      and [`aea-user`](../user-image/README.md#publish))
- [ ] I have pushed the latest packages to the registry.
- [ ] I have uploaded the latest `aea` to PyPI.
- [ ] I have uploaded the latest plugins to PyPI.

## Further comments

Write here any other comment about the release, if any.

### CODE REVIEW

File: packages/fetchai/skills/aries_alice/handlers.py [hunk 1]
```diff
@@ -120,13 +84,15 @@ def handle(self, message: Message) -> None:
...
+            self.context.logger.info("Received message content:" + repr(content))
             if "@type" in content:
                 strategy = cast(Strategy, self.context.strategy)
                 details = self._handle_received_invite(content)
                 self.context.behaviours.alice.send_http_request_message(
                     method="POST",
-                    url=strategy.admin_url + ADMIN_COMMAND_RECEIVE_INVITE,
+                    url=strategy.admin_url
+                    + ADMIN_COMMAND_RECEIVE_INVITE
+                    + "?auto_accept=true",
```
Comment (5A11) [inline packages/fetchai/skills/aries_alice/handlers.py:95]: Why is this needed?

File: packages/fetchai/skills/aries_faber/handlers.py [hunk 2]
```diff
@@ -101,11 +98,28 @@ def _send_default_message(self, content: Dict) -> None:
...
+        """Register DID on the ledger."""
+        strategy = cast(Strategy, self.context.strategy)
+        self.context.behaviours.faber.send_http_request_message(
+            method="POST",
+            url=strategy.admin_url
+            + ADMIN_COMMAND_REGISTGER_PUBLIC_DID
+            + f"?did={self.did}",
+            content="",
+        )
+
```
Comment (5A11) [inline packages/fetchai/skills/aries_faber/handlers.py:111]: What is the diff between this and "_register_did" below?

Comment (solarw) [reply packages/fetchai/skills/aries_faber/handlers.py:130]: updates

File: packages/fetchai/skills/aries_faber/behaviours.py [hunk 3]
```diff
@@ -85,6 +85,7 @@ def act(self) -> None:
         strategy = cast(Strategy, self.context.strategy)
         if strategy.is_searching:
             query = strategy.get_location_and_service_query()
+            print(100000000000000000000000, query)
```
Comment (5A11) [inline packages/fetchai/skills/aries_faber/behaviours.py:88]: What is this?

Comment (solarw) [reply packages/fetchai/skills/aries_faber/behaviours.py]: fixed

**Review by 5A11 — APPROVED** [review]
lgtm
